src/Model/Storage/SystemResource.php

```diff
@@ -41,6 +42,16 @@ class SystemResource implements IStorage
 	 */
 	public function get(string $filename)
 	{
+		$parts = parse_url($filename);
+		if (!empty($parts['scheme']) && !empty($parts['host'])) {
+			$curlResult = DI::httpRequest()->get($filename);
+			if ($curlResult->isSuccess()) {
+				return $curlResult->getBody();
+			} else {
+				return "";
+			}
+		}
```

Comment:
No, no no no, no no no no no.

What is going on here?

Reply:
You look thrilled :grimacing: 

This is some kind of bug fix. We used the `SystemResource` class before to fetch external content. But: This hadn't worked for anything else than files although we provided HTTP links for this class.

Reply:
Then we should stop providing URLs to this class instead of defiling its intent. We can add an Exception for this, we'll quickly find the places where URLs shouldn't be passed to this class. 😈
```suggestion
		$parts = parse_url($filename);
		if (!empty($parts['scheme']) && !empty($parts['host'])) {
			thrown new \Exception('URL wrongly passed to SystemResource', 500);
		}
```

Reply:
I created a new class.